pvlib/iotools/psm3.py

```diff
@@ -42,8 +43,12 @@ def get_psm3(latitude, longitude, api_key, email, names='tmy', interval=60,
         PSM3 API parameter specifing year or TMY variant to download, see notes
         below for options
     interval : int, default 60
```

Comment:
```suggestion
    interval : int, {60, 5, 15, 30}
```

Reply:
I don't love the out of order presentation, but numpydoc says put the default first. https://numpydoc.readthedocs.io/en/latest/format.html#sections